### PR TITLE
transforms/subsetGoogleFonts: Added fontDisplay option

### PR DESCRIPTION
**Pull request by Munter**

### CODE REVIEW

File: lib/transforms/subsetGoogleFonts.js [hunk 1]
```diff
@@ -373,6 +382,24 @@ module.exports = function (options) {
                         fontStyleBundle = new AssetGraph.Css(assetGraph.resolveAssetConfig({
                             text: source
                         }));
+
+                        if (fontDisplay) {
+                            fontStyleBundle.parseTree.nodes.forEach(function (atRule) {
+                                var declarations = atRule.nodes;
```
Comment (papandreou) [inline lib/transforms/subsetGoogleFonts.js]: Hmm, this won't detect `@font-face` rules nested inside `@media` queries.

Also, we should only dive into `@font-face` rules here in the code that follows? (`atRule.name === 'font-face'`)

Comment (Munter) [reply lib/transforms/subsetGoogleFonts.js]: Yeah, this is pretty native and will probably only work for Google fonts, where I can make a lot of assumptions about the stylesheet. I think the generic case might be a bit harder

Comment (papandreou) [reply lib/transforms/subsetGoogleFonts.js]: Fair :)

File: lib/transforms/subsetGoogleFonts.js [hunk 2]
```diff
@@ -133,11 +133,20 @@ var fontFormatUA = {
...
+    'fallback',
+    'optional'
+];
+
 module.exports = function (options) {
     options = options || {};
     var subsetPerPage = !!options.subsetPerPage;
     var inlineSubsets = !!options.inlineSubsets;
     var inlineCss = !!options.inlineCss;
+    var fontDisplay = validFontDisplayValues.includes(options.fontDisplay) && options.fontDisplay;
```
Comment (papandreou) [inline lib/transforms/subsetGoogleFonts.js]: The build fails with node 4 due to the use of `Array#includes`

Comment (Munter) [reply lib/transforms/subsetGoogleFonts.js]: Fixed and rebased

**Pull request merged**